Library/Formula/vegeta.rb

```diff
@@ -0,0 +1,28 @@
+require "language/go"
+
+class Vegeta < Formula
+  homepage "https://github.com/tsenart/vegeta"
+  url "https://github.com/tsenart/vegeta/archive/v5.5.2.tar.gz"
+  sha1 "a0b2d4c4bca0023a8407a819f64c3f96e1ebe3bc"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/bmizerany/perks" do
+    url "https://github.com/bmizerany/perks.git",
+      :revision => "6cb9d9d729303ee2628580d9aec5db968da3a607"
+  end
+
+  def install
+    mkdir_p buildpath/"src/github.com/tsenart/"
+    ln_s buildpath, buildpath/"src/github.com/tsenart/vegeta"
+    ENV["GOPATH"] = buildpath
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    system "go", "build", "-o", "vegeta"
+    bin.install "vegeta"
+  end
+
+  test do
+    pipe_output("#{bin}/vegeta attack -duration=1s -rate=1", "GET http://localhost/")
```

Comment:
It's testing that it doesn't immediately fail doing something slightly more complex than `--help` or `--verbose`.